network-api/networkapi/buyersguide/templates/bg_base.html

```diff
@@ -16,7 +16,6 @@
     <meta property="og:image:height" content="628" />
     <meta property="og:url" content="{{request.scheme}}://{{request.get_host}}{{request.path}}" />
     <link rel="stylesheet" href="/_css/buyers-guide.compiled.css">
-    <link rel="stylesheet" href="https://code.cdn.mozilla.net/fonts/fira.css">
```

Comment:
looks like this slipped in from a different branch, can you undo these three `fira.css` changes in this branch?

Reply:
sure. will do that.